test/system/005-info.bats

```diff
@@ -33,7 +33,7 @@ RunRoot:
     run_podman info --format=json
 
     expr_nvr="[a-z0-9-]\\\+-[a-z0-9.]\\\+-[a-z0-9]\\\+\."
-    expr_path="/[a-z0-9\\\/.]\\\+\\\$"
+    expr_path="/[a-z0-9\\\-\\\/.]\\\+\\\$"
```

Comment:
LGTM but just as a matter of readability (and I realize there is some difference of opinion about this) my eyes find it much easier to recognize dash-as-a-character when it's at the beginning or end of the character class: 
```
   expr_path="/[a-z0-9\\\/.-]\\\+\\\$"       (with apologies for so many backslashes)
```
To me that makes it clear that the author explicitly meant the dash, and not a range. But I'm fine with this either way. Thanks for the fix.

Reply:
@edsantiago That's good to know. I'm happy to change it but I see a whole lot of checks running below. ;)

Reply:
Yeah - don't annoy the CI gods. It's fine as it is, especially with all the backslashes to make it stand out. Thanks again.

Reply:
I was counting my backslashes very carefully... :cowboy_hat_face: